packages/block-editor/src/components/list-view/block.js

```diff
@@ -298,6 +298,7 @@ function ListViewBlock( {
 							} }
 							disableOpenOnArrowDown
 							__experimentalSelectBlock={ updateSelection }
+							context={ 'list-view' }
```

Comment:
Should there also be a `context={ 'block-toolbar' }` somewhere else in the codebase?

I also wonder if this should be documented somewhere.

Reply:
> Should there also be a context={ 'block-toolbar' } somewhere else in the codebase?

Context param is currently only used for List View. It _could_ be used elsewhere but if it isn't provided then it's `undefined` so we should be all good. I didn't feel it was necessary update all instances of `<BlockSettingsDropdown>` with a prop that isn't required. Do you feel that I should do that? I'm trying to keep this PR's scope down where possible.

> I also wonder if this should be documented somewhere.

I've now documented this within the component's README.